test/rating.js

```diff
@@ -48,8 +48,7 @@ describe('A track in Google Music', function () {
 
   describe('when \'thumbs up\'-ed', function () {
     browserUtils.execute(function resetRating () {
-      // DEV: Warning this will skip to next track
-      window.googleMusic.rating.toggleThumbsDown();
+      window.googleMusic.rating.resetRating();
```

Comment:
Now that we have `resetRating` as a documented method, we need to test it as well =/